templates/base_email.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
I am not sure what purpose this base template serves if none of the other emails use it

also not sure what <title> does in an email either

Reply:
it had the bootstrap CSS, that's all. thought we might want to use it for all templates. but good point, until then it belongs in the same template.